WeatherToday/Sources/Data/NetworkAPI/RequestConfigurations/WeatherApiRequestConfigurations.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  WeatherApiRequestConfigurations.swift
+//  WeatherToday
+//
+//  Created by Taher on 10/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Alamofire
+import Foundation
+
+enum WeatherApiRequestConfigurations: RequestConfiguration {
+
+    case get(cityName: String)
+
+    var baseURL: String {
+        Configuration.baseWeatherURL
+    }
+
+    var endpoint: String {
+        "data/2.5/weather"
+    }
+
+    var method: HTTPMethod {
+        switch self {
+        case .get(_):
+            return HTTPMethod.get
```

Comment:
FYI, we can write it like this
```suggestion
        case .get(_): return HTTPMethod.get
```

Reply:
resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/d19faa8fbaec66b5f882dd4d38f85f4a4ca738c1)